tests/utils/command-utils.go

```diff
@@ -0,0 +1,37 @@
+package utils
+
+import (
+	"bytes"
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+	"runtime"
+)
+
+func RunScriptFromBaseDir(command string, args []string, envVars ...string) (string, string, error) {
+	_, filename, _, _ := runtime.Caller(0)
+	dir := path.Join(path.Dir(filename), "..", "..")
+	return RunCommand("bash", append([]string{fmt.Sprintf("%s/%s", dir, command)}, args...), envVars...)
```

Comment:
Perhaps a '-c' in here (see https://github.com/opendevstack/ods-core/issues/331#issuecomment-566449448) would allow for type -P as expected?

Reply:
I would favor the PR #338 and disregard -P for now, but also these scripts need `chmod +x`. There is a lot of improvements needed which could go as a separate issue.